coffee-model/coffee-model-base/pom.xml

```diff
@@ -60,6 +60,25 @@
 			<artifactId>hibernate-jpamodelgen</artifactId>
 			<version>${hibernate.jpamodelgen.version}</version>
 			<scope>provided</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.xml.bind</groupId>
+					<artifactId>jaxb-api</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>javax.activation</groupId>
+					<artifactId>javax.activation-api</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<!-- hibernate-jpamodelgen -> javax.xml.bind:jaxb-api és javax.activation:javax.activation-api helyett -->
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
mivel ez csak a generatornak kell ami `<scope>provided</scope>` igy ezeknek is annak kell lennie

Reply:
kiszedtem a generátort és a vele járó plusz dolgokat a dependencyk közül, már csak a maven-compile-pluginban jelennek meg